gsd/version.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2020 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+
+
+__version__ = "2.1.1"

```

Comment:
Add this file to the bumpversion configuration.
Also add it to the list in `CMakeLists.txt` so it is included in the CMake builds for testing.